SampleConnector.sln

```diff
@@ -13,9 +13,7 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SampleConnectorCommon", "Sa
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SampleConnector", "SampleConnector\SampleConnector.csproj", "{D986EE01-3E67-4D55-9229-873F38CA36BE}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SampleConnectorWorkerRole", "SampleConnectorCloudService\SampleConnectorWorkerRole\SampleConnectorWorkerRole.csproj", "{1BEDEA96-626A-4FED-9644-D4BC880B4CBC}"
-EndProject
-Project("{CC5FD16D-436D-48AD-A40C-5A424C6E3E79}") = "SampleConnectorCloudService", "SampleConnectorCloudService\SampleConnectorCloudService.ccproj", "{A7CE2440-7099-42FC-B5D2-56B0B8AA8834}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SampleConnectorWebJob", "SampleConnectorWebJob\SampleConnectorWebJob.csproj", "{706416EF-AE0F-4954-BDA0-58CA8F0DA77B}"
```

Comment:
Is it possible to make this subdirectory of webapi itself ?

Reply:
No it is configured that way. Add a project as a webjob